tests/app/dao/test_services_dao.py

```diff
@@ -506,6 +507,50 @@ def test_fetch_weekly_historical_stats_separates_weeks(notify_db, notify_db_sess
     assert ret[-1].count == 1
 
 
+def test_fetch_monthly_historical_stats_separates_weeks(notify_db, notify_db_session, sample_template):
+    notification_history = functools.partial(
+        create_notification_history,
+        notify_db,
+        notify_db_session,
+        sample_template
+    )
+    _before_start_of_financial_year = notification_history(created_at=datetime(2016, 3, 31))
+    start_of_financial_year = notification_history(created_at=datetime(2016, 4, 1))
+    start_of_summer = notification_history(created_at=datetime(2016, 6, 20))
```

Comment:
we should test that sept 30, 23:00 counts in October. And March 31 11:30 resolves to April 1